physical_properties/analysis/analyse.py

```diff
@@ -1,204 +1,236 @@
-import json
+"""A script to analyze the outputs of benchmarking each force field of interest against
+the test set of physical properties and produce relevant plots and tables.
+"""
 import os
-import re
-import sys
-from enum import Enum
-from io import StringIO
-from collections import defaultdict
-from typing import Tuple, Dict
+from typing import Dict, List, Tuple
 
-import numpy as np
-
-import matplotlib
+import numpy
+import pandas
+import seaborn
 from matplotlib import pyplot
-from openforcefield.topology import Molecule, Topology
-from openforcefield.typing.engines.smirnoff import ForceField
-from openforcefield.utils import UndefinedStereochemistryError
+from openff.evaluator import properties
+from openff.evaluator.datasets import PhysicalPropertyDataSet
+
+seaborn.set_context("paper", font_scale=2.2)
 
-from propertyestimator import unit
-from propertyestimator.utils.serialization import TypedJSONDecoder
+pyplot.rc('text', usetex=True)
```

Comment:
This line adds a hard dependency on latex (not conda-installable), which prevented me from running the script. Removing it didn't seem to have a major effect. Maybe consider making it a soft dependency.

```suggestion
try:
    pyplot.rc('text', usetex=True)
except:
    pass
```